lib/snjsTypes.ts

```diff
@@ -31,7 +31,8 @@ export enum ComponentAction {
   SaveError = 'save-error',
   ThemesActivated = 'themes-activated',
   KeyDown = 'key-down',
-  KeyUp = 'key-up'
+  KeyUp = 'key-up',
+  Click = 'click'
```

Comment:
Do we need to distinguish between right click and left click?

Reply:
Currently we'd only need to get left-clicks, right-click if used anywhere else than on a note cell should fire the browser's default menu and that's all. But right-clicks fire a different event, the 'contextmenu' event, so I think we don't need to distinguish